config/routes.rb

```diff
@@ -2,7 +2,9 @@
   devise_for :users
   root to: "pages#home"
 
-  resources :tournaments, only: %i[index new create show]
+  resources :tournaments, only: %i[index new create show] do
+    resources :players, only: %i[create]
```

Comment:
Just to confirm, we don't need a 'new' route defined for players? Is it not a new page that we have to send an empty player to